### PR TITLE
App freezes when we press enter with empty pin in lock screen

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/589/app-freezes-when-we-press-enter-with-empty-pin-in-lock-screen

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt [hunk 1]
```diff
@@ -17,3 +17,5 @@ data class PinAuthenticated(val data: Any?) : PinEntryEvent() {
 }
 
 data class PinVerified(val result: PinVerificationMethod.VerificationResult) : PinEntryEvent()
+
+object PinEntryDoneClicked : PinEntryEvent()
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt:21]: Can we add an analytics event name for this?

**Review by vinaysshenoy — CHANGES_REQUESTED** [review]
PR looks good, left a comment. Please take a look?

File: app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt:21]: Currently, we are not verifying or do anything on this click. So I was planning on adding this once we start doing something? 🤔

Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/security/pin/PinEntryEvent.kt:21]: Ah, good point. It looks good then.

**Review by vinaysshenoy — APPROVED** [review]

**Pull request merged**